src/state/storage-provider.ts

```diff
@@ -112,12 +119,14 @@ export class S3StorageProvider implements IStorageProvider {
 
     public async put(contents: string): Promise<void> {
         try {
-            const s3client = new S3({ credentials: this.credentials });
+            /**
+             * I was getting an ACL error here in govcloud.
+             */
+            const s3client = new S3({ credentials: this.credentials, region: this.region });
             const putObjectRequest: PutObjectRequest = {
                 Bucket: this.bucketName,
                 Key: this.objectKey,
                 Body: contents,
-                ACL: 'bucket-owner-full-control',
```

Comment:
this not supported by us-gov? if so -> we'll have to figure out something to keep this for commercial but take it out for partition